torch/csrc/jit/codegen/cuda/scheduler.cpp

```diff
@@ -96,9 +96,10 @@ bool scheduleFusion(Fusion* fusion, const at::ArrayRef<c10::IValue> inputs) {
     auto out = out_val->as<TensorView>();
 
     // Merge all dimensions because we're only supporting pointwise
-    while (out->nDims() > 1) {
-      out->merge(-2, -1);
-    }
+    // Real reductions aren't supposed to reach here
+    // This is a workaround to handle trivial reductions, i.e. size-1 reductions
+    mergeReduction(out);
```

Comment:
I don't think this should hurt, but I think we should be safe without it, no?

Reply:
Yes, we should be fine without merging the reductions. Removed in the recent commit and added a new test `FusionTrivialReduction3` to cover multiple trivial reductions in one `TensorView`.